src/log/log.c

```diff
@@ -2121,6 +2121,19 @@ __wt_log_scan(WT_SESSION_IMPL *session, WT_LSN *lsnp, uint32_t flags,
         WT_SET_LSN(&end_lsn, lastlog, 0);
         WT_ERR(__wt_fs_directory_list_free(session, &logfiles, logcount));
     }
+
+    if (start_range_lsnp != NULL) {
+        /* Check to see if the user set offset is in allignment,
```

Comment:
This comment should be formatted such that Check is on a second line like:
```
/*
 * Check
```
Additionally you should try and fill the 100 character limit, I'm surprised that ./s_all didn't pick that one up. Finally the word align only has one l.

Reply:
Is that a 100 character limit for each comment block?

Reply:
In our style the line itself is up to a maximum of 100 characters and we should try use as much of that as possible.

Reply:
Okay no worries